### DISCUSSION TITLE
Create list of good-first-issues

### DISCUSSION THREAD

**Discussion by jquorning** [alire-project/alire#1264] [Ideas]
I would like to see a list of *beginner-friendly-problems* like minor fixes, refactors, small improvement that could be done without much trouble.

This is to lower the threshold for the ones new to Ada program maintenance.
In other words; A long list of problems easy to fix.

I myself has made some minor contributions to Alire, but I could do much more if the problems were served in pieces.

**Comment by mosteo** [comment]
We have a `cost: 0-trivial` label that can serve to identify simple issues. We are not applying those consistently though as sometimes is difficult to evaluate how complex a change will be.
